readme.md

```diff
@@ -1,2 +1,67 @@
 React Material Icons
 ====================
+
+This package includes all [Material Icons](https://material.io/resources/icons/?style=baseline) as reusable react components
```

Comment:
I would write "react" as "React", because it's the right way to write it.

Reply:
All names have been updated